### PR TITLE
Fixing pagination for Active Tests and Past Order Results Data Tables under routine tests tab on patient chart

### PR DESCRIPTION
**Pull request by Daphne210**
https://github.com/openmrs/openmrs-esm-laboratory/assets/89844768/c01dc8d5-ba47-4eac-93f8-677263c8de4f

Screenshot (131)

### CODE REVIEW

**Comment by pirupius** [comment]
Hi @Daphne210 please update the description of the PR with the work done and screenshots/video demo

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "main": "src/index.ts",
     "source": true,
     "scripts": {
-        "start": "openmrs develop",
+        "start": "openmrs develop --backend http://194.163.171.253:8282",
```
Comment (jabahum) [inline package.json:10]: remove this url

**Comment by Daphne210** [comment]
> Hi @Daphne210 please update the description of the PR with the work done and screenshots/video demo

hey @pirupius this has been added

File: package.json (hunk 1 above)
Comment (Daphne210) [reply package.json]: > remove this url

Hey @jabahum this has been removed

**Comment by jabahum** [comment]
@Daphne210 is this inclusive of the referrals table as well

**Comment by Daphne210** [comment]
> @Daphne210 is this inclusive of the referrals table as well

The referrals table is okay

**Comment by jabahum** [comment]
@Daphne210 thanks

**Review by pirupius — APPROVED** [review]
Thanks @Daphne210

**Pull request merged**